aries-site/src/layouts/content/WCAGRuleSummary.js

```diff
@@ -24,6 +20,10 @@ const STATUS_MAP = {
     label: 'Conditional',
     icon: <CircleInformation alt="" />,
   },
+  'AAA failed': {
+    label: 'AAA Failed',
```

Comment:
```suggestion
    label: 'Does not meet AAA',
```

OR

```suggestion
    label: 'AAA not met',
```

OR

```suggestion
    label: 'AAA not achieved',
```

Reply:
I like 'AAA not met'

Reply:
I thought we were keeping the same phrasing with the "failed" so if users read this they arent thinking whats the different between 'not met' vs failed

Reply:
Yeah Matt and I chatted about this and were feeling like the trade off between 'not met' and 'failed' was consistent language vs potential alarm when people see the word 'failed'.

Felt like it might be more beneficial to not cause alarm, soften the wording to something like not met rather than have consistent wording by using the term 'failed'.

Reply:
gotcha I wonder then `AAA not achieved` also sounds like like a little softer as well maybe we can think about the naming but yeah id be good with either `AAA not achieved` or  'AAA not met'

Reply:
^ Agree I like both `AAA not achieved` or `AAA not met`